Storage/rest/advanced_hypermedia.h

```diff
@@ -34,15 +34,21 @@ SOFTWARE.
 #include "hypermedia.h"
 #include "sfinae.h"
 
-CURRENT_STRUCT(AdvancedHypermediaRESTRecordResponse) {
+namespace current {
+namespace storage {
+namespace rest {
+
+CURRENT_STRUCT_T(AdvancedHypermediaRESTRecordResponse) {
+  CURRENT_FIELD(success, Optional<bool>);
```

Comment:
Strictly speaking, `null` is an explicit falsy value, so either explicit `true`, or no such `success` field at all in the scheme. But, of course, `success` for collections could be just ignored on the client.